docs/examples/test-app/deploy.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+# Copyright 2020 The Kubernetes Authors.
+# SPDX-License-Identifier: Apache-2.0
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# Get a Kubernetes cluster ready before deploying the application
+
+# Deploy the Application CRD with the controller on the Kubernetes cluster
+make deploy
+
+# Install the test CRD to the cluster
+kubectl apply -f test_crd.yaml
```

Comment:
We may have to wait for the crd to be registered.

Reply:
I will add a readln to confirm the CRD is registered.